src/librustc/diagnostics.rs

```diff
@@ -168,6 +168,11 @@ match x {
 ```
 "##,
 
+E0010: r##"
+The value of static and const variables must be known at compile time, and they
```

Comment:
‘const variable’ is an oxymoron. This should probably just read something like ‘The value of constants (declared with either `const` or `static`) must be known…’